uwsgi/public.wsgi

```diff
@@ -37,6 +37,8 @@ cache.make_cache("blob", 500, 3600)
 cache.make_cache("blob_schema", 50, 24 * 60 * 60)
 cache.make_cache("blob_version", 500, 60)
 
+cache.make_cache("rules", 20, 30)
```

Comment:
Yeah. I think I might increased the size though. My fresh morning brain thinks there's probably more than 20 combinations of product+buildTarget, and it shouldn't take up too much memory anyways (especially not compared to the blob cache.